pkg/account/component.go

```diff
@@ -194,7 +187,11 @@ func (c *component) UpdateAccount(ctx context.Context, user api.AccountRepresent
 	}
 
 	//store the API call into the DB
-	_ = c.reportEvent(ctx, "UPDATE_ACCOUNT", database.CtEventRealmName, realm, database.CtEventUserID, userID, database.CtEventUsername, username)
+	errEvent := c.reportEvent(ctx, "UPDATE_ACCOUNT", database.CtEventRealmName, realm, database.CtEventUserID, userID, database.CtEventUsername, username)
```

Comment:
Maybe we could also have a method for those line as it seems to be very similar every time ? 
What do you think of it ?

Reply:
DONE